helm/values-prod.yaml

```diff
@@ -69,10 +69,15 @@ microservice-chart:
     COSMOS_URI: https://pagopa-p-weu-afm-marketplace-cosmos-account.documents.azure.com:443/
     ISSUER_RANGE_TABLE: 'pagopapweuafmsaissuerrangetable'
     AMOUNT_THRESHOLD: '0'
+    ELASTIC_APM_SERVICE_NAME: pagopa-afm-calculator
+    ELASTIC_APM_APPLICATION_PACKAGES: it.gov.pagopa.afm.calculator
+    ELASTIC_APM_SERVER_URL: http://quickstart-apm-http.elastic-system.svc:8200
+    ELASTIC_APM_LOG_SENDING: 'true'
```

Comment:
@pasqualespica I use this experimental feature of APM instead of filebeat. Is it ok?
([doc](https://www.elastic.co/guide/en/observability/master/apm-agent-log-sending.html))

Reply:
[oki for now](https://www.elastic.co/guide/en/apm/agent/java/current/config-logging.html#config-log-sending)
We could see [apm filebeat cfg](https://www.elastic.co/guide/en/observability/master/ecs-reformatting-with-filebeat.html) too in to next sprint